org.filezillaproject.Filezilla.json

```diff
@@ -86,8 +87,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.filezilla-project.org/libfilezilla/libfilezilla-0.45.0.tar.xz",
-                    "sha512": "304838e8cf6e644378c0727bc8b9d2f87287a646db4cb765a04d156717f8c233b8a34085049e4893ec345a1a1d576f0ba33df705dee022e16ff433cddd926ce0"
+                    "url": "http://deb.debian.org/debian/pool/main/libf/libfilezilla/libfilezilla_0.49.0.orig.tar.xz",
```

Comment:
what happened to upstream source?

Reply:
As far as I can tell, they changed to some kind of cdn/mirror infrastructure, that doesn't like permanent links.

E.g. https://lib.filezilla-project.org/download.php generates this for my download and works right now. https://dl1.cdn.filezilla-project.org/libfilezilla/libfilezilla-0.50.0.tar.xz?h=WiRqnDNSvKz4Wp8-j07CZw&x=1745508709

It will probably be dead in some minutes, that's at least what my testing yesterday lead to.

Trying to use the link without the params, will cause us to download the webpage, but named like the archive, which then errors out due to `xz` being confused.

Reply:
there is some level of irony here.

Reply:
https://release-monitoring.org/project/15391/ and https://release-monitoring.org/project/809/ also seem very unhappy due to those changes (maybe just in parts)